ground/uavobjgenerator/uavobjectparser.cpp

```diff
@@ -404,6 +404,12 @@ QString UAVObjectParser::parseXML(QString& xml, QString& filename)
         // Calculate size
         calculateSize(info);
 
+        // Buffer on flight side is 256 bytes
+        // Additional overhead is 2+4+2+2+1+1=12
+        if(info->numBytes > 244)

```

Comment:
@kubark42 As icee mentioned there isn't a common header between the two locations and I can't really see a clean way to do so. I'll take another look at this later today and implement the requested modifications re: overhead. Thanks @icee and @kubark42.